src/tiles/list/tile-list.ts

```diff
@@ -0,0 +1,326 @@
+/* eslint-disable lit/no-invalid-html */
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { SortParam } from '@internetarchive/search-service';
+import DOMPurify from 'dompurify';
+import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import '../../mediatype-icon';
+
+@customElement('tile-list')
+export class TileList extends LitElement {
+  @property({ type: Object }) model?: TileModel;
+
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam?: SortParam;
+
+  render() {
+    return html`
+      <div id="list-line" class="${this.classSize}">
+        <div id="list-line-left">
+          <div id="thumb">${this.img()}</div>
+          ${this.iconLeft()}
+        </div>
+        <div id="list-line-right">
+          <div id="title">${DOMPurify.sanitize(this.model?.title ?? '')}</div>
```

Comment:
question:
should we add `dompurify` in all title and creator data?

Reply:
Yes.